python/cog/server/http.py

```diff
@@ -195,7 +197,10 @@ async def predict_idempotent(
                     )
                 ]
             )
-
+        if runner.is_busy():
```

Comment:
Why did this get added? Isn't this already handled by `_predict`?

Reply:
mistake